daliuge-engine/dlg/named_port_utils.py

```diff
@@ -461,16 +461,6 @@ def optionalEval(x):
             return ast.literal_eval(content) if len(content) > 0 else None
 
         reader = optionalEval
-    elif input_parser is DropParser.UTF8:
```

Comment:
Why has this been remove? There is no reason defined in the PR description.

Reply:
Yes, that is missing in the description, but there is a separate commit for it: There is already an `elif input_parser is DropParser.UTF8` a few lines further up, thus this one will never be executed anyway.